_posts/2018-11-20-mongodb-europe-018.md

```diff
@@ -0,0 +1,411 @@
+  <img src="/img/2018-11-08-mongodb-europe-2018/create-mongodb-atlas-cluster.png" width="100%">
+</div>
+
+Start your own cluster: [https://cloud.mongodb.com](https://cloud.mongodb.com){:target="_blank" rel="noopener noreferrer"}
+
+### Cloud
+MongoDB Atlas is a cross-platform database. 
+You can run it on Amazon Web Services, Google Cloud or Microsoft Azure. 
+It provides you an availability map that shows you which users are served from where an what latency should be expected.
+<div style="text-align: center;">
+  <img src="/img/2018-11-08-mongodb-europe-2018/global-clusters.png" width="100%">
+</div>
+
+### Scalability
+When your database grows it is easy to scale up or out.
+You can scale up by increasing the size of the instance on which your database runs.
+Scaling out is done by the process of sharding.
+Here we are storing data of the same collection across multiple smaller partitions so we can distribute these partitions over multiple machines and increase read-write performance.
+This way you do not run into the limitations of a single server.
+
+To ensure that MongoDB stores data equally across shards you need the right strategy of choosing a partition key.
+
+### High availability 
+When a primary node goes down, a new primary is chosen immediately by the system of voting.
+All nodes vote on who should become the new primary. 
+The node with the majority of the votes becomes the new primary.
+A general guideline is to have a replica set that consists of one primary node and at least two secondary nodes.
+To ensure maximum uptime the procedure to recover from instance failures is completely automated.
+
+
+# MongoDB University
+(General Session Keynote)
+
+With [MongoDB University](https://university.mongodb.com/){:target="_blank" rel="noopener noreferrer"}, Mongo has its own platform for online courses.
+A lot of them are available for free.
+You can pick out courses according to your needs or profession.
+There are training tracks for Developers, DBA's and everyone else...
+The courses are ideal to get you started or to deepen your knowledge when you want to be more advanced.
+If you want you can even get certified!
+I speak from my own experience when I say that the University platform is great to work with, and the courses are very well taught.
+Find all available resources here: [https://university.mongodb.com/](https://university.mongodb.com/){:target="_blank" rel="noopener noreferrer"}.
+
+
+
+# Compass Aggregation Pipeline Builder
+(General Session Keynote)
+
+Recently the aggregation pipeline builder was introduced in [MongoDB Compass](https://www.mongodb.com/products/compass){:target="_blank" rel="noopener noreferrer"}.
+This allows you to create an aggregation pipeline step by step.
+And that makes it easy to debug the pipeline along the way.
+Let's see an example:   
+Suppose I have a collection which contains documents that represent a person.
+Here are two examples of elements in the collections:
```

Comment:
This sentence feels forced. Why not add ", like this:" to end of the previous one?